app/routes/_index.tsx

```diff
@@ -16,28 +18,21 @@ type LoaderData = {
   apiBaseURL: string;
 }
 
-export const loader: LoaderFunction = async ({ context }): Promise<LoaderData> => {
+export const loader: LoaderFunction = async ({ context } : LoaderFunctionArgs): Promise<LoaderData> => {
+  //const url: string = import.meta.env.VITE_ALIGNMENT_FEED_BASE_URL;
+  //return { apiBaseURL: url };
```

Comment:
I think it would be good to remove commented out earlier versions of things rather than leaving them in to keep the codebase clean and easy to iterate on.